src/cgtop/cgtop.c

```diff
@@ -56,6 +56,12 @@ typedef struct Group {
         uint64_t io_input_bps, io_output_bps;
 } Group;
 
+typedef enum PidsCount {
+        COUNT_USERSPACE_PROCESSES,
+        COUNT_ALL_PROCESSES,
+        COUNT_PIDS,
```

Comment:
The fact that the order matters now, should *really* be documented in a comment here. Otherwise the next person looking at this will end up changing this by accident

Reply:
It is documented in the commit message :grin:

The existing commit has been merged, does a code comment here warrant a new commit & PR?

Reply:
Yeah, just do a new PR, having the comment in the commit message makes it hard to find, in this case it's good to have it in the source code as well

Reply:
btw: → https://systemd.io/CODING_STYLE/#commenting

but yeah, commits and PRs are free. never hesitate to send a follow-up. always welcome